spec/core/ics-004-channel-and-packet-semantics/UPGRADES.md

```diff
@@ -388,11 +390,11 @@ function chanUpgradeTry(
     // get current sequence on this channel
     // if the counterparty sequence is greater than the current sequence, we fast forward to the counterparty sequence
     // so that both channel ends are using the same sequence for the current upgrade
-    // if the counterparty sequence is less than the current sequence, then either the counterparty chain is out-of-sync or
+    // if the counterparty sequence is less than or equal to the current sequence, then either the counterparty chain is out-of-sync or
     // the message is out-of-sync and we write an error receipt with our own sequence so that the counterparty can update
     // their sequence as well. We must then increment our sequence so both sides start the next upgrade with a fresh sequence.
     currentSequence = provableStore.get(channelUpgradeSequencePath(portIdentifier, channelIdentifier))
-    if counterpartySequence >= currentSequence {
+    if counterpartySequence > currentSequence {
         provableStore.set(channelUpgradeSequencePath(portIdentifier, channelIdentifier), counterpartySequence)
```

Comment:
I feel it may be useful to reassign currentSequence as well, in case the spec gets refactored later, this would need to be accounted for